sdk/eventhub/Azure.Messaging.EventHubs.Processor/tests/BlobsCheckpointStore/BlobsCheckpointStoreTests.cs

```diff
@@ -21,7 +23,18 @@ public class BlobsCheckpointStoreTests
         [Test]
         public void ConstructorRequiresBlobContainerClient()
         {
-            Assert.That(() => new BlobsCheckpointStore(null), Throws.InstanceOf<ArgumentException>());
+            Assert.That(() => new BlobsCheckpointStore(null, Mock.Of<EventHubsRetryPolicy>()), Throws.InstanceOf<ArgumentNullException>());
```

Comment:
(extreme) nit:  You can collapse this to `Throws.ArgumentNullException` since we know that is the exact type of exception.